src/module/actor/dataModelClasses/OseDataModelCharacterEncumbrance.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * A class to handle character encumbrance.
+ */
+ export default class OseDataModelCharacterEncumbrance {
+  static encumbranceCap = 1600;
```

Comment:
Suggest renaming this to `defaultEncumbranceCap`

Reply:
Maybe `baseEncumbranceCap` to fit with other default values?

Reply:
Makes sense to me.